### DISCUSSION TITLE
Date Range Picker

### DISCUSSION THREAD

**Discussion by VishalDudeja** [rsuite/rsuite#2471] [Q&A]
I'm using rsuit  v5.8.0", Date Range Picker in React.js project,  and importing CSS using '/node_modules/rsuite/dist/rsuite.css''
But the imported rsuit css is disturbing project's overall global css 
Is there any way to prevent this? 

Any help would be highly appreciated.
Thanks.
![rsuit css issue](https://user-images.githubusercontent.com/73440646/165068406-bd225172-d094-4073-90a9-8d8bcc6a1558.png)

**Comment by SevenOutman** [comment]
You may have to [use rsuite Less style](https://rsuitejs.com/guide/customization/) and modify `@enable-css-reset` variable to be `false`.

```less
@import 'rsuite/styles/index.less';

@enable-css-reset: false;
```

**Reply by VishalDudeja** [reply]
Hey,
Thanks for reply.
Importing 'rsuite/styles/index.less' and adding @enable-css-reset: false;  in css file isn't helping.
Even the DateRangePicker is not looking good after using it.
![image](https://user-images.githubusercontent.com/73440646/165268637-7317c5d1-f403-4517-bc23-566de836f840.png)

**Reply by SevenOutman** [reply]
> Importing 'rsuite/styles/index.less' and adding @enable-css-reset: false; in css file isn't helping.

Did you setup Less support in your project or only added an import line into your css file?

**Comment by VishalDudeja** [comment]
Hey,
Thanks for reply again.
I didn't setup Less support before importing.
I'll do that and let you know if that works.

Thanks
